nilearn/plotting/js_plotting_utils.py

```diff
@@ -120,8 +120,8 @@ def get_iframe(self, width=None, height=None):
         if height is None:
             height = self.height
         escaped = escape(self.html, quote=True)
-        wrapped = '<iframe srcdoc="{}" width={} height={}></iframe>'.format(
-            escaped, width, height)
+        wrapped = ('<iframe srcdoc="{}" width={} height={} '
+                   'frameBorder="0"></iframe>').format(escaped, width, height)
         return wrapped
 
     def get_standalone(self):
```

Comment:
The `__str__` method does the same thing, so one can use `str()` to get this, obviating the need for this method.

Reply:
Does `_repr_html_` method name (defined below, line #131) have some special meaning for Jupyter Notebooks?  If not, then the method `get_iframe()` already does the job.

Reply:
> Does _repr_html_ method name (defined below, line #131) have some special meaning for Jupyter Notebooks?

yes, this is why they are displayed inline in jupyter notebooks

Reply:
> The __str__ method does the same thing, so one can use str() to get this, obviating the need for this method.

this is to be more explicit, if we want a full html document or an iframe that can be inserted somewhere.

Reply:
any changes to this part of nilearn should be done in another PR since this one doesn't touch it

Reply:
Okay, I'll delete these as well.